tsconfig.json

```diff
@@ -14,19 +17,23 @@
     "resolveJsonModule": true,
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
-    "typeRoots": ["./src/types/custom", "node_modules/@types"],
+    "typeRoots": [
+      "./src/types/custom",
+      "node_modules/@types"
+    ],
     "rootDir": ".",
     "allowJs": true,
     "removeComments": true,
     "esModuleInterop": true,
-    "emitDecoratorMetadata": true
+    "emitDecoratorMetadata": true,
   },
   "include": [
     "src/**/*",
-    "tests/**/*"
+    "test/**/*",
+    "node_modules/ts-jest/globals.d.ts"
```

Comment:
Is there a legit reason something from node_modules is in the include list? It seems to work just fine without it.